### PR TITLE
Added build script for kafka-python

### PR DESCRIPTION
**Pull request by atharv-phadnis**
Package Name: **kafka-python**
Package Version: **2.0.2**
Package URL: [kafka-python](https://github.com/dpkp/kafka-python)

Tested on: **UBI 8.5**
Build Errors: **None**
Test Errors: **None**

### CODE REVIEW

**Comment by seth-priya** [comment]
@Ryan-Dmello please review

File: k/kafka-python/kafka-python_ubi_8.5.sh [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+git checkout $PACKAGE_VERSION
+
+export KAFKA_VERSION=0.11.0.3
+
+source $HOME_DIR/$PACKAGE_NAME/travis_java_install.sh
+source $HOME_DIR/$PACKAGE_NAME/build_integration.sh
+
+cd $HOME_DIR/$PACKAGE_NAME
+if ! tox -e py36; then
+	echo "------------------$PACKAGE_NAME:install_or_test_fails---------------------"
```
Comment (Ryan-Dmello) [inline k/kafka-python/kafka-python_ubi_8.5.sh:47]: Tox command is going to perform package testing only. Please refer file tox.ini for details.

For installation use following command, after line 45:

`pip3 install ./`

Please update the wordings, after adding installation command.

**Review by Ryan-Dmello — CHANGES_REQUESTED** [review]
@atharv-phadnis other than the comment I mentioned, build script looks ok to me

File: k/kafka-python/kafka-python_ubi_8.5.sh (hunk 1 above)
Comment (Ryan-Dmello) [reply k/kafka-python/kafka-python_ubi_8.5.sh:47]: Also package is installable via pip3 command. 
`pip3 install kafka-python`

Please check if build script is actually required

**Comment by atharv-phadnis** [comment]
@Ryan-Dmello Made the required changes, please have a look at the script.

Was getting random test errors using `pip3 install .`, so went with the classic `python3 setup.py install`.

**Comment by Ryan-Dmello** [comment]
Latest changes looks ok

**Review by Ryan-Dmello — APPROVED** [review]

**Pull request merged**